pzp/qml/area.qml

```diff
@@ -1,111 +1,249 @@
 <!DOCTYPE qgis PUBLIC 'http://mrcc.com/qgis.dtd' 'SYSTEM'>
-<qgis version="3.43.0-Master" readOnly="0" styleCategories="LayerConfiguration|Symbology|Labeling|Fields|Forms" labelsEnabled="1">
+<qgis labelsEnabled="1" version="3.42.2-Münster" simplifyDrawingTol="1" minScale="0" maxScale="0" styleCategories="AllStyleCategories" hasScaleBasedVisibilityFlag="0" simplifyDrawingHints="1" readOnly="0" simplifyMaxScale="1" symbologyReferenceScale="-1" autoRefreshMode="Disabled" autoRefreshTime="0" simplifyAlgorithm="0" simplifyLocal="1">
```

Comment:
Could we preserve the exported categories for this QML?

Not sure if this is exactly the case, but if I remember correctly, I think that I ended up exporting only `LayerConfiguration|Symbology|Labeling|Fields|Forms` to avoid other categories to mess things up in the new layers.

In case you want to stick to `AllCategories`, I would double check `Custom Properties` and `Geometry Options` categories, to make sure they don't interfere with the expected configuration for the new Area layer.

Reply:
Good idea, not sure what's the best place for that, though.
One place could be when we load the `.qml` files in the code.

Reply:
even better would be to check that in the pipeline